docs/edgeql/functions/converters.rst

```diff
@@ -23,3 +23,24 @@ the functions below allow more options for such conversions.
 
         db> SELECT to_json('{"hello": "world"}')['hello'];
         {'world'}
+
+.. eql:function:: std::to_str(json: json, fmt: OPTIONAL str={}) -> str
+
+    :index: stringify dumps
+
+    Return string representation of the input value.
+
+    When converting :eql:type:`json`, this type converter can take an
+    extra argument ``fmt := 'pretty'``. If ``'pretty'`` formatting is
+    specified, then the JSON value is serialized with extra whitespace
+    to make it more human-readable.
```

Comment:
I don't like this phrasing. I'd change it to "The optional \*fmt\* parameter can be set to 'pretty' to produce pretty-formatted JSON string."

Reply:
I've changed the wording.